### DISCUSSION TITLE
File name template

### DISCUSSION THREAD

**Discussion by Hulk9165** [Tyrrrz/YoutubeDownloader#414] [Q&A]
Hello,
I really like this program and when i download files from youtube, i would like to add in the filename the "upload_date" tag.
When i look in the "File name templates" settings, there nothing.
Is there a way to add it?
Thanks

**Comment by Tyrrrz** [comment]
It's not supported because it's not available on playlist videos

**Comment by Hulk9165** [comment]
OK, thanks for your answer.
